### DISCUSSION TITLE
teoria relazionale - 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#70] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/20f8e9ff-064d-4e2d-b739-2f3a37866b49)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-04 173823](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/4eaba549-a485-4a11-9d57-d22ffc70dc7c)
![Immagine 2024-02-04 173838](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/a8ade56c-a60d-4d22-a81a-30774537199f)
![Immagine 2024-02-04 174051](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/e2facd10-c3e6-40c6-857c-fcb51ab33ae0)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/cbab43d1-717c-4784-b0a1-5553b2f2ee1a)
